data/darktableconfig.xml.in

```diff
@@ -1309,7 +1309,7 @@
     <type>bool</type>
     <default>true</default>
     <shortdescription>middle mouse button zooms to 200%</shortdescription>
-    <longdescription>when clicking the middle mouse button to zoom, the zoom level will cycle through 100%, 200% and then back to fit the screen. otherwise it switches between fit to screen and 100% and the 'ctrl' key can be used to control the zoom level.</longdescription>
+    <longdescription>If enabled, the zoom level will cycle between 100%, 200% and fit to viewport on middle mouse clicks. If disabled, it will toggle between viewport size and 100%, and the Ctrl key can be used to control the zoom level.</longdescription>
```

Comment:
Please no capital letters here. You're confused with your work on en@truecase :)

Reply:
Apart from that I'm ok with the wording.

Reply:
> Please no capital letters here. You're confused with your work on en@truecase :)

Oh indeed! I've been so exhausted lately that it's hard for me to concentrate now and I'm prone to making these mistakes due to a loss of focus. Sorry, fixed.